samples/graphene-hello/spawn/hello.c

```diff
@@ -0,0 +1,16 @@
+#include <stdio.h>
+
+int main(int argc, char* argv[], char* envp[]) {
+  puts("Hello world!\n\n\033[0;33mCommandline arguments:\033[0m");
+
+  for (int i = 0; i < argc; ++i)
+    puts(argv[i]);
+
+  puts("\n\033[0;33mEnvironment variables:\033[0m");
+
+  for (int i = 0; envp[i]; i++) {
```

Comment:
Now that I tested it, I think we should only print the env var from the manifest (with getenv).

Reply:
I opted for a middle way and decided to use the old approach we used in EGo: Filter everything with "EDG_" and "MARBLE_PREDEFINED", so that any user defined variables (+ the ones defined in the Graphene manifest) are visible.

Honestly, I think showing every environment variable is not too bad in general. It's looks cluttered, yes, but gives the developer an insight into the existing environment of the Graphene.

However, you can check out my new approach for this. Guess that should be a compromise...? :)

Reply:
I don't like the filtering code here, but agree that showing every env var may be good. How about printing only the first n chars?